.github/workflows/scripts/build-images.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
[nit] I just realized it doesn't include the license header. Could you extend the `license` Makefile target to check `./.github` too and add licenses to script files there, please?

Reply:
Good point, added.